examples/calibrate_tsqrtP.py

```diff
@@ -0,0 +1,322 @@
+                fl.plot(
+                    abs(s["t":int_range]),
+                    color="black",
+                    label="integrated slice",
+                )
+                plt.ylabel(r"$\sqrt{P}$ / $\sqrt{W}$")
+                plt.text(
+                    int_range[0] * 1e6 - 1,
+                    0.25,
+                    r"$t_{pulse} \sqrt{P_{tx}} = %f \mathrm{μs} \sqrt{\mathrm{W}}$"
+                    % (beta_v_t["t_pulse", j].item() / 1e-6),
+                )
+            # }}}
+        # {{{ show what we observe -- how does β vary with the programmed pulse length
+        fl.basename = None  # reset so all amplitudes are on same plots
+        fl.next(r"Measured $\beta$ vs A * $t_{pulse}$")
+        beta_v_t.rename("t_pulse", "$A t_{pulse}$")
+        beta_v_t.name(r"$\beta$")
+        beta_v_t[
+            "$A t_{pulse}$"
+        ] *= amplitude  # we only t_pulse to be multiplied by amplitude for plotting purposes only!
+        fl.plot(
+            (beta_v_t.C / 1e-6),
+            color=thiscolor,
+            label="amplitude = %f" % amplitude,
+        )
+        beta_v_t[
+            "$A t_{pulse}$"
+        ] /= amplitude  # need to divide back out for the determination of the coefficients below
+        psd.gridandtick(plt.gca())
+        beta_v_t.rename("$A t_{pulse}$", "t_pulse")
+        # }}}
+        # {{{ Identify captures that don't increase in beta - don't use
+        decreasing_idx = np.nonzero(~(np.diff(beta_v_t.data) > 0))[0]
+        if (
+            len(decreasing_idx) > 0
+        ):  # beta doesn't always increase with increasing pulse length
+            fl.plot(
+                beta_v_t["t_pulse", : decreasing_idx[-1] + 1],
+                "x",
+                color="r",
+                label="can't use these",
+            )
+            # throw out betas following a faulty pulse
+            beta_v_t = beta_v_t["t_pulse", decreasing_idx[-1] + 1 :]
+        # }}}
+        plt.axhline(  # the linear threshold is the threshold above which beta is linear
+            y=linear_threshold / 1e-6,
+            color=thiscolor,
+            label=f"linear threshold for amp={amplitude}",
+        )
+        # {{{ flip data so beta is on x axis now and t_pulse is the y axis
+        t_v_beta = (
+            beta_v_t.shape.alloc(dtype=np.float64)
+            .rename("t_pulse", r"$\beta$")
+            .setaxis(r"$\beta$", beta_v_t.data)
+        )
+        t_v_beta.data[:] = (
+            beta_v_t["t_pulse"].copy() / 1e-6
+        )  # because our ppg wants μs
+        t_v_beta.set_units("μs").set_units(r"$\beta$", "s√W")
+        # use as temp for ultimate coeff
+        # }}}
+        # {{{ Determine linear and nonliear coefficients
+        c_nonlinear = t_v_beta[
+            r"$\beta$":(None, linear_threshold)
+        ].C  # make copy to allow the next
+        #                                                               line not to cause the data to
```

Comment:
comment is weirdly formatted.  Leave the hash at the start, but put all the letters on the second like so that it's aligned under make -- but the entire rest of the comment here.  Then highlight and `gq`